tools/cldr-apps/src/main/java/org/unicode/cldr/web/util/JSONObject.java

```diff
@@ -0,0 +1,51 @@
+package org.unicode.cldr.web.util;
+
+import com.google.gson.Gson;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.TreeMap;
+
+/* shim */
+public class JSONObject implements JSONString {
```

Comment:
It's a little surprising that this general-purpose code needs to be built from scratch. Is there a viable path for migrating this to a dependency on something not written by us?

Reply:
yes, using Google's gson directly.

Reply:
- consider the web/api/ classes, where we just use plain objects (with jackson serialization almost completely hidden)
- JsonUtil.gson() is now used universally in the cldr-apps module, you can see examples of it searching for gson().
- cldr-code uses Gson separately in two instances.

Reply:
Should our home-grown JSONObject be deprecated?

Reply:
This appears related to https://unicode-org.atlassian.net/browse/CLDR-18513